app/views/priority.scala.html

```diff
@@ -36,5 +36,13 @@
             </div>
         </main>
     }
+
+	@maybeTelemetryUrl.map { telemetryUrl => {
+        <script type="module">
+			const image = new Image();
+			image.src = `@Html(telemetryUrl)&path=${window.location.pathname}`;
+		</script>
+		}
+	}
```

Comment:
Would it be possible to move this into a reusable Twirl component to avoid the repetition? I'm also curious, do we need to the `type="module"` attribute?

Reply:
Great idea!  I've moved the snippet into a reusable twirl component in the next commit.

If we set the attribute `type="module"`, the code inside will be executed as a ES module and so the variables/constants created there will not be in the global scope.  I think it could minimize the risk of accidentally breaking the existing JS code (though it is highly unlikely there is really a constant `image` in the global scope).

Reply:
Good point, I hadn't thought of that